bears/python/PEP8Bear.py

```diff
@@ -28,15 +28,15 @@ def run(self, filename, file,
             local_pep8_config: bool = False,
             ):
         """
-        Detects and fixes PEP8 incompliant code. This bear will not change
+        Detects and fixes PEP8 incompliant code.This bear will not change
```

Comment:
Please do not remove the space after the full stop, that is, you need not change anything in this line. Thanks!

Reply:
okay can you tell me what exactly I need to do to solve this issue

Reply:
Please add back the white space between `code.`  and `This`. Change:
`Detects and fixes PEP8 incompliant code.This bear will not change` to `Detects and fixes PEP8 incompliant code. This bear will not change`

Reply:
okay, 
but  i think it doesn't solve the issue only